data_model/src/lib.rs

```diff
@@ -341,7 +343,60 @@ pub enum Value {
     /// [`struct@Hash`]
     Hash(Hash),
     /// Block
-    Block(BlockValue),
+    Block(BlockValueWrapper),
+}
+
+/// Cross-platform wrapper for `BlockValue`.
+#[cfg(not(target_arch = "aarch64"))]
+#[derive(
+    AsRef,
+    Clone,
+    Debug,
+    Decode,
+    Deref,
+    Deserialize,
+    Encode,
+    Eq,
+    From,
+    Into,
+    IntoSchema,
```

Comment:
Shouldn't we manually implement `IntoSchema` to hide the presence of the `BlockValueWrapper` from the generated schema?
It's seems like our inner stuff.

Reply:
Oh. A good point. I'll take a look. Thanks!

Reply:
This is not mission-critical, but I'm marking the PR as a draft until this is resolved.